### PR TITLE
Nitpick: Pass secret as file

### PR DESCRIPTION
**Pull request by Deeds67**
According to this site, passing secrets as env vars through k8s is not allowed: https://kyverno.io/policies/other/disallow-secrets-from-env-vars/disallow-secrets-from-env-vars/

It looks like there are security concerns when exposing secrets in this way, since env vars are exposed in logging frameworks & are accessible by anyone who has access to docker in the VM. 

Our current implementation passes the secret as a volume (which is the recommended way), however we then read this file in the `.start.sh` script and then pass it as an environment variable to the application. We are therefore just doing a work-around to avoid the policy.

This change in this PR is that the application now reads the secret directly from the file, instead of passing it as an env var.

### CODE REVIEW

File: .tool-versions [hunk 1]
```diff
@@ -0,0 +1 @@
+nodejs 20.17.0
```
Comment (Deeds67) [inline .tool-versions:1]: I couldn't get `asdf` to read the legacy `.node-version` file, even after adding `legacy_version_file = yes` to my `~/.asdfrc`. The only way I got it to work was using the new `.tool-versions` format. 

@manuelpuchta Would it be OK if we switch from the legacy file to the new one, or should we just keep around both?

File: app/config/config.server.ts [hunk 2]
```diff
@@ -8,14 +10,17 @@ interface Config {
 
 let instance: Config | undefined = undefined;
 
+const oidcClientSecretFilePath = "/etc/brak-idp-secrets/BRAK_IDP_OIDC_CLIENT_SECRET"
```
Comment (Deeds67) [inline app/config/config.server.ts:13]: I considered passing this hardcoded path as ENV var, but in my experience I've never changed this value, so I just hardcoded it. @manuelpuchta What do you think?

File: app/config/config.server.ts [hunk 3]
```diff
@@ -8,14 +10,19 @@ interface Config {
 
 let instance: Config | undefined = undefined;
 
+const oidcClientSecretFilePath =
```
Comment (Deeds67) [inline app/config/config.server.ts:13]: I considered passing this hardcoded path as ENV var, but in my experience I've never changed this value, so I just hardcoded it. @manuelpuchta  What do you think?

Comment (manuelpuchta) [reply app/config/config.server.ts:13]: This is great, let's ship it and test it.

ℹ️ If it is working, let us update that part on the platform docs [guidelines](https://platform-docs.prod.ds4g.net/user-docs/how-to-guides/ci-cd/seal-secret#use-sealedsecret-as-environment-variable) "Use SealedSecret as environment variable".

File: .tool-versions (hunk 1 above)
Comment (manuelpuchta) [reply .tool-versions:1]: As discussed, let's switch to `.tool-versions`, also in the pipeline config, please.

**Review by manuelpuchta — APPROVED** [review]

**Pull request merged**